preview-docs/GraphRoles.md

```diff
@@ -2,20 +2,23 @@
-4. Introduce a way to assign roles to graph API keys. We would make the **Consumer**, **Observer**, **Contributor**, and **Graph Admin** roles available for assignment on API key creation. (Existing graph API keys would act as **Contributor**s.) Users can create and view keys with roles that have power equal to or less than their role on the graph.
-5. Introduce a setting on graphs that flags them as "visible" or "hidden" to the members of that graph's org by default. Right now, all graphs in the org are visible to all members of the org. If you mark a graph as "hidden" to your org, only users with explicit overridden roles on the graph (as described in "2") or who are **Org Admin**s can see it. (While **Org Admin**s will have full read and write access to hidden graphs, hidden graphs that they do not have explicit overridden roles on may still be hidden by default from graph lists in the UI unless explicitly requested.) **Graph Admin**s and **Org Admin**s can control whether graphs are hidden.
+4. Introduce a way to assign roles to graph API keys. We would make the **Consumer**, **Observer**, **Contributor**, and **Graph Admin** roles available for assignment on API key creation. (Existing graph API keys continue to support the same operations they currently support.) **Graph Admins** can create and view graph keys with any role.
+5. Introduce a setting on graphs that flags them as "visible" or "hidden" to the members of that graph's org by default. Right now, all graphs in the org are visible to all members of the org. If you mark a graph as "hidden" to your org, only users with explicit overridden roles on the graph (as described in "2") or who are **Org Admin**s can see it. **Graph Admin**s and **Org Admin**s can control whether graphs are hidden.
+
 
-### Other considerations
+**We intend to release these new features on Tuesday, January 19th.** For full details on precisely what each role will be able to do after this feature is release, you can view a [preview of the docs](https://deploy-preview-1080--studio-docs.netlify.app/bdocs/studio/org/members/).
 
-This plan does not yet tackle variant-specific tokens, but we think there's value to that and will likely end up adding that as well sooner rather than later.
+The following visible changes will occur when the features are released:
 
-Having variant-specific tokens will allow us to create contributor tokens for protected variants, which would save us from needing an added Deployer role in between Contributor and Graph Admin at some point.
+- All **Admin**s will now be called **Org Admin**s.
+- All **Contributor**s will be changed to **Graph Admin**s. In addition to what they could do before, these users can now delete graphs and manage graph-specific permissions. If this is more than these users should be able to do, you may want to change them to being **Contributor**s instead.
```

Comment:
```suggestion
- Because the new **Contributor** role is substantially different from current behaviors, and because the new **Graph Admin** role is very close in permissions to to the old **Contributor** role, all previous **Contributor**s will be changed to **Graph Admin**s. In addition to what they could do before, these users can now delete graphs and manage graph-specific permissions. If these new permissions are more than these users should be able to do, you will be able to change them to being (the new) **Contributor**s instead. If for some reason this change will be problematic, please contact your account team immediately.
```